cpr/CMakeLists.txt

```diff
@@ -45,3 +45,8 @@ add_library(${CPR_LIBRARIES}
 message(STATUS "Using CURL_LIBRARIES: ${CURL_LIBRARIES}.")
 target_link_libraries(${CPR_LIBRARIES}
     ${CURL_LIBRARIES})
+    
+install(TARGETS ${CPR_LIBRARIES}
```

Comment:
oh, overlooked that. That would indeed be problematic.
The curl headers and libraries are installed even without the patch (tried with master) since curl is
simply added as subdir, so it is not possible at the moment to not install it.